Improvement/refactor json parsing (#90)
NationalSecurityAgency/pelz: README.md

     * URI syntax must currently comply with RFC 8089 and RFC 1738 Section 3.1.

> * key\_id_len : int

Reviewer: ```suggestion

```

Author: Fixed with 1ac5ce1.

>     * Integer specifying the length of the key_id URI.

Reviewer: ```suggestion

```

Author: Fixed with 1ac5ce1.

* data : string of characters

>     * Base64 encoded version of the key to be wrapped or unwrapped.

Reviewer: ```suggestion
    * Base64 encoded data to be processed based on request type.
```

Author: Fixed with 1ac5ce1.

Examples:

JSON Request for AES Key Wrap
* {"key_id": "file:~/pelz/test/key1.txt", "request_type": 1, "data": "YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4\n"}
* {"key_id": "pelz://localhost/7000/fake_key_id", "request_type": 1, "data": "YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4\n"}

JSON Request for AES Key Unwrap
* {"key_id": "file:~/pelz/test/key1.txt", "request_type": 2, "data": "BtIjIgvCaVBwUi5jTOZyIx2yJamqvrR0BZWLFVufz9w=\n"}
* {"key_id": "pelz://localhost/7000/fake_key_id", "request_type": 2, "data": "BtIjIgvCaVBwUi5jTOZyIx2yJamqvrR0BZWLFVufz9w=\n"}

#### Response JSON Key and Values
* key_id : string of characters
    * URI for the key location (key identifier).
    * The key_id specified in the JSON request will be included in the JSON response.
* data : string of characters

>     * Base-64 encoded, output data.

Reviewer: ```suggestion
    * Base-64 encoded, output data based on request type.
```
